canonical/snapcraft.io
# feat: Move settings into publisher app

## Done
- Moved publisher settings view into the publisher React application

## How to QA
- Go to https://snapcraft-io-4847.demos.haus/<SNAP_NAME>/settings
- Check that changes to the form can be made and saved

## Testing

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-14632
